"Unexpected registry type 3, expected REG_QWORD" when getting graphics cards on Windows

Hi again @dbwiddis,

a user who ran code to get his graphics cards got the following Exception:
```
SystemInfo systemInfo = new SystemInfo();
HardwareAbstractionLayer hardware = systemInfo.getHardware();
List<GraphicsCard> graphicsCards = hardware.getGraphicsCards();
// ...
```
```
Caused by: java.lang.RuntimeException: Unexpected registry type 3, expected REG_QWORD
	at com.sun.jna.platform.win32.Advapi32Util.registryGetLongValue(Advapi32Util.java:1228) ~[jna-platform-5.17.0.jar:5.17.0 (b0)]
	at com.sun.jna.platform.win32.Advapi32Util.registryGetLongValue(Advapi32Util.java:1199) ~[jna-platform-5.17.0.jar:5.17.0 (b0)]
	at com.sun.jna.platform.win32.Advapi32Util.registryGetLongValue(Advapi32Util.java:1174) ~[jna-platform-5.17.0.jar:5.17.0 (b0)]
	at oshi.hardware.platform.windows.WindowsGraphicsCard.getGraphicsCards(WindowsGraphicsCard.java:85) ~[oshi-core-6.9.2.jar:6.9.2]
	at oshi.hardware.platform.windows.WindowsHardwareAbstractionLayer.getGraphicsCards(WindowsHardwareAbstractionLayer.java:87) ~[oshi-core-6.9.2.jar:6.9.2]
```
Upon some research and debugging, I figured that the registry field `Computer\HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Class\{4d36e968-e325-11ce-bfc1-08002be10318}\0001` is only checked for existance, then it's being read as a long regardless of which data type the registry key has:
<img width="879" height="145" alt="Image" src="https://github.com/user-attachments/assets/91d8ddd0-bc58-4b7c-9aa5-b2043303a31a" />
However, for unknown reasons the user seems to have a `REG_BINARY` field there instead (registry type 3):
<img width="968" height="274" alt="Image" src="https://github.com/user-attachments/assets/36c53ed2-8350-48d7-bc88-fa4e57f651fb" />
I would suggest implementing this code more flexibly and defensively. I see that you already did something like this for handling the `MEMORY_SIZE` value in the `else if` block. Maybe `QW_MEMORY_SIZE` should be handled the same way? It's definitely preferred to not fail the entire method because trying to read the `QW_MEMORY_SIZE` `vram` value fails but other things might all succeed regarding building the `cardList` further down in the `getGraphicsCards()` method.

**EDIT:**
Indeed, the user is using [`Parallels`](https://www.parallels.com) to run Windows on a Mac and the software somehow likes to store the graphics cards fields as `REG_BINARY`:

<img width="2390" height="2035" alt="Image" src="https://github.com/user-attachments/assets/c3adbf35-1c3e-4c3a-8f5a-ed1913d08c04" />

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,7 @@
 # 6.9.3 (in progress)
 
-* Your contribution here!
+##### Bug fixes / Improvements
+* [#3064](https://github.com/oshi/oshi/pull/3064): Handle binary registry values in WindowsGraphicsCard - [@dbwiddis](https://github.com/dbwiddis).
 
 # 6.9.0 (2025-09-18), 6.9.1 (2025-10-18), 6.9.2 (2025-12-20)
 

--- a/oshi-core/src/main/java/oshi/hardware/platform/windows/WindowsGraphicsCard.java
+++ b/oshi-core/src/main/java/oshi/hardware/platform/windows/WindowsGraphicsCard.java
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020-2025 The OSHI Project Contributors
+ * Copyright 2020-2026 The OSHI Project Contributors
  * SPDX-License-Identifier: MIT
  */
 package oshi.hardware.platform.windows;
@@ -81,11 +81,13 @@ final class WindowsGraphicsCard extends AbstractGraphicsCard {
                 String versionInfo = RegistryUtil.getStringValue(WinReg.HKEY_LOCAL_MACHINE, fullKey, DRIVER_VERSION);
                 long vram = 0L;
 
-                if (Advapi32Util.registryValueExists(WinReg.HKEY_LOCAL_MACHINE, fullKey, QW_MEMORY_SIZE)) {
-                    vram = Advapi32Util.registryGetLongValue(WinReg.HKEY_LOCAL_MACHINE, fullKey, QW_MEMORY_SIZE);
-                } else if (Advapi32Util.registryValueExists(WinReg.HKEY_LOCAL_MACHINE, fullKey, MEMORY_SIZE)) {
-                    Object genericValue = Advapi32Util.registryGetValue(WinReg.HKEY_LOCAL_MACHINE, fullKey,
-                            MEMORY_SIZE);
+                String memKey = Advapi32Util.registryValueExists(WinReg.HKEY_LOCAL_MACHINE, fullKey, QW_MEMORY_SIZE)
+                        ? QW_MEMORY_SIZE
+                        : (Advapi32Util.registryValueExists(WinReg.HKEY_LOCAL_MACHINE, fullKey, MEMORY_SIZE)
+                                ? MEMORY_SIZE
+                                : null);
+                if (memKey != null) {
+                    Object genericValue = Advapi32Util.registryGetValue(WinReg.HKEY_LOCAL_MACHINE, fullKey, memKey);
                     if (genericValue instanceof Long) {
                         vram = (long) genericValue;
                     } else if (genericValue instanceof Integer) {

